### PR TITLE
Update to server page, cloudinary, asset_serve_url

### LINKED ISSUE #290: Update /server page
Update the entire page with the translations attached to this [brief](https://docs.google.com/document/d/1onVw-6_3zNzGXSyny8L8unC90QdVH-DJ5ZPEwdm0iNw/edit)

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Updated the /server page per docs provided in the [brief](https://docs.google.com/document/d/1onVw-6_3zNzGXSyny8L8unC90QdVH-DJ5ZPEwdm0iNw/edit)
- Added the {{ ASSET_SERVER_URL }} pointing to cloudinary and added this to non svg, pdf and js files
- Update the version of Django and global_nav

## QA

1. download this branch
2. ./run the site 
3. review the [server page](http://0.0.0.0:8010/server)
4. compare to the doc in the brief

## Issue / Card

Fixes #290

### CODE REVIEW

**Comment by pmahnke** [comment]
@zprood can you look this over as well... I question weather all the links back to the UK site are correct, particularly the contact-us?

File: templates/cloud/index.html [hunk 1]
```diff
@@ -28,11 +28,11 @@
     fill: #fff; }
 
   .p-takeover--private-cloud-economics {
-    background-image: url(https://assets.ubuntu.com/v1/13fdbc5b-graph.png), linear-gradient(45deg, #171717 0%, #181818 9%, #262626 34%, #2D2D2D 67%, #383838 88%, #2E2E2E 100%, #363636 100%);
+    background-image: url({{ ASSET_SERVER_URL }}13fdbc5b-graphpng), linear-gradient(45deg, #171717 0%, #181818 9%, #262626 34%, #2D2D2D 67%, #383838 88%, #2E2E2E 100%, #363636 100%);
     background-color: #191919; }
      @media (min-width: 768px) {
       .p-takeover--private-cloud-economics{
-      background-image: url(https://assets.ubuntu.com/v1/13fdbc5b-graph.png), linear-gradient(45deg, #171717 0%, #181818 9%, #262626 34%, #2D2D2D 67%, #383838 88%, #2E2E2E 100%, #363636 100%);
+      background-image: url({{ ASSET_SERVER_URL }}13fdbc5b-graphpng), linear-gradient(45deg, #171717 0%, #181818 9%, #262626 34%, #2D2D2D 67%, #383838 88%, #2E2E2E 100%, #363636 100%);
```
Comment (anthonydillon) [inline templates/cloud/index.html]: Missing the . On the file extension

File: templates/cloud/index.html [hunk 2]
```diff
@@ -163,31 +163,31 @@ <h2 class="p-muted-heading u-align-text--center">我们的部分OPENSTACK客户<
...
         <li class="p-inline-images__item">
-          <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/379ae6d1-Yahoo_Japan_logo.png" alt="Yahoo japan">
+          <img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}379ae6d1-Yahoo_Japan_logo.png" alt="Yahoo japan">
         </li>
         <li class="p-inline-images__item">
           <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/b7693339-logo-bloomberg.svg" alt="Bloomberg">
         </li>
         <li class="p-inline-images__item">
-          <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/4ef05fb7-bestbuy.jpg" alt="Best Buy">
+          <img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}4ef05fb7-bestbuyjpg" alt="Best Buy">
```
Comment (anthonydillon) [inline templates/cloud/index.html]: Missing the . On the file extension

File: templates/cloud/index.html [hunk 3]
```diff
@@ -163,31 +163,31 @@ <h2 class="p-muted-heading u-align-text--center">我们的部分OPENSTACK客户<
...
-          <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/192f9755-partners-logo-nec-tagline.png" alt="NEC">
+          <img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}192f9755-partners-logo-nec-tagline.png" alt="NEC">
         </li>
         <li class="p-inline-images__item">
-          <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/377e12ed-logo-ntt-communications.png" alt="NTT">
+          <img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}377e12ed-logo-ntt-communications.png" alt="NTT">
         </li>
         <li class="p-inline-images__item">
-          <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/5482b658-liveperson-logo.jpg" alt="LivePerson">
+          <img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}5482b658-liveperson-logojpg" alt="LivePerson">
```
Comment (anthonydillon) [inline templates/cloud/index.html]: Missing the . On the file extension

File: templates/cloud/index.html [hunk 4]
```diff
@@ -265,13 +265,13 @@ <h2>完全托管的OpenStack云</h2>
...
-          <img src="https://assets.ubuntu.com/v1/d56d0b96-Biznet-Gio.png" alt="Biznet Gio">
+          <img src="{{ ASSET_SERVER_URL }}d56d0b96-Biznet-Gio.png" alt="Biznet Gio">
         </li>
         <li class="p-inline-images__item p-inline-images__item--smaller">
-          <img src="https://assets.ubuntu.com/v1/ff4e18ac-mimedia_logo_lrg.png" alt="Mimedia">
+          <img src="{{ ASSET_SERVER_URL }}ff4e18ac-mimedia_logo_lrg.png" alt="Mimedia">
         </li>
         <li class="p-inline-images__item p-inline-images__item--smaller">
-          <img src="https://assets.ubuntu.com/v1/507a0979-Voidbridge.jpg" alt="Voidbridge">
+          <img src="{{ ASSET_SERVER_URL }}507a0979-Voidbridgejpg" alt="Voidbridge">
```
Comment (anthonydillon) [inline templates/cloud/index.html]: Missing the . In the file extension

File: templates/cloud/index.html [hunk 5]
```diff
@@ -281,7 +281,7 @@ <h2>完全托管的OpenStack云</h2>
   </div>
 </section>
 
-<section class="p-strip--image u-no-background--small is-deep is-bordered" style="background-image: url('https://assets.ubuntu.com/v1/1d2ab6ba-suru-background.png'); background-size: 100% 100%;">
+<section class="p-strip--image u-no-background--small is-deep is-bordered" style="background-image: url('{{ ASSET_SERVER_URL }}1d2ab6ba-suru-backgroundpng'); background-size: 100% 100%;">
```
Comment (anthonydillon) [inline templates/cloud/index.html]: File extension

File: templates/cloud/index.html [hunk 6]
```diff
@@ -322,7 +322,7 @@ <h2>在公有云上使用Ubuntu</h2>
...
       <figure id="ubuntu1-pie" class="ubuntu1-pie"><svg width="308" height="308" viewBox="0 0 310 310"><g transform="translate(155,155)"><path d="M9.429780353434619e-15,-154A154,154 0 1,1 -90.5189288530409,124.58861713374188L-45.25946442652045,62.29430856687094A77,77 0 1,0 4.7148901767173095e-15,-77Z" class="ubuntu" style="stroke: rgb(174, 167, 159); stroke-width: 1; stroke-dasharray: 3, 3;"></path><path d="M-90.5189288530409,124.58861713374188A154,154 0 0,1 -2.8289341060303857e-14,-154L-1.4144670530151929e-14,-77A77,77 0 0,0 -45.25946442652045,62.29430856687094Z" class="other" style="stroke: rgb(174, 167, 159); stroke-width: 1; stroke-dasharray: 3, 3;"></path><text y="-5" class="ubuntu-label"><tspan x="85.32943725585938" dy="0">ubuntu</tspan><tspan x="88.422607421875" dy="20">(60%)</tspan></text><text y="-5" class="other-label"><tspan x="-133.09738159179688" dy="0">other</tspan><tspan x="-132.77914428710938" dy="20">(40%)</tspan></text></g></svg></figure>
-      <script src="https://assets.ubuntu.com/v1/3c99518b-d3-version3.5.6.min.js"></script>
+      <script src="{{ ASSET_SERVER_URL }}3c99518b-d3-version35.6.min.js"></script>
```
Comment (anthonydillon) [inline templates/cloud/index.html]: File name does not match

File: templates/cloud/index.html [hunk 7]
```diff
@@ -506,7 +506,7 @@ <h2 class="p-muted-heading u-align--center" style="max-width: none">Juju上的
         <img src="https://assets.ubuntu.com/v1/683950fd-logo-ibm.svg" alt="IBM">
       </li>
       <li class="p-inline-images__item">
-        <img src="https://assets.ubuntu.com/v1/a2a68c52-logo-juniper-160.png?w=160" alt="Juniper">
+        <img src="{{ ASSET_SERVER_URL }}a2a68c52-logo-juniper-160png?w=160" alt="Juniper">
```
Comment (anthonydillon) [inline templates/cloud/index.html]: File extension

File: templates/internet-of-things/index.html [hunk 8]
```diff
@@ -38,7 +38,7 @@ <h2>用于软件定义万物的应用程序商店</h2>
   <div class="row">
     <div class="col-12 u-hide--small">
 
-      <img src="https://assets.ubuntu.com/v1/30a4ee8e-branded+store+snaps.jpg" alt="Snap store screenshot">
+      <img src="{{ ASSET_SERVER_URL }}30a4ee8e-branded+store+snapsjpg" alt="Snap store screenshot">
```
Comment (anthonydillon) [inline templates/internet-of-things/index.html]: File extension

File: templates/server/index.html [hunk 9]
```diff
@@ -13,543 +13,480 @@
...
+            <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/4d6054f9-logo-cisco.svg" width="144" alt="Cisco" />
+          </li>
+          <li class="p-inline-images__item">
+            <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/731aab6c-logo-amd.svg" width="144" alt="AMD" />
+          </li>
+          <li class="last-item p-inline-images__item">
+            <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/4efbd291-Arm_logo_blue_cropped.svg" width="144" alt="Arm" />
+          </li>
+        </ul>
+        <p class="u-align--center u-vertically-spaced"><a href="https://certification.ubuntu.com/">查看所有认证硬件&nbsp;›</a></p>
```
Comment (anthonydillon) [inline templates/server/index.html]: External class

File: templates/server/index.html [hunk 10]
```diff
@@ -13,543 +13,480 @@
...
+            <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/eed716e9-logo-openstack.svg" width="88" alt="OpenStack" />
+          </li>
+          <li class="p-inline-images__item">
+            <img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}c228b4e9-logo-likewise.png" width="144" alt="Likewise" />
+          </li>
+          <li class="last-item p-inline-images__item">
+            <img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}64493bba-logo-openbravo.png" width="144" alt="Openbravo" />
+          </li>
+        </ul>
+        <p class="u-align--center u-vertically-spaced"><a href="https://partners.ubuntu.com/programmes/software">查看所有认证软件&nbsp;›</a></p>
```
Comment (anthonydillon) [inline templates/server/index.html]: External link

File: templates/server/index.html [hunk 11]
```diff
@@ -13,543 +13,480 @@
...
-        <h3 class="p-card__title">联系我们</h3>
-        <p class="p-card__content">请联系我们的技术支持团队说明您的服务器要求。</p>
-        <p class="p-card__content"><a class="p-button--positive p-link--external" href="https://pages.ubuntu.com/contact-us-in-chinese.html" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Contact Us', 'eventLabel' : 'cn.ubuntu.com - server', 'eventValue' : undefined });">联系我们</a>
-
-        </p>
+    <div class="row">
+      <div class="col-6">
+        <p>Ubuntu Advantage 服务计划可帮您快速解决问题、直接向 Ubuntu 专家寻求帮助，并可使用 Ubuntu 系统管理软件包 Landscape 进行高效管理。</p>
+        <p>Landscape 让您可以像管理一台计算机那样轻松管理数千台 Ubuntu 计算机，使 Ubuntu 桌面计算机、服务器和云实例的管理更具成本效益。</p>
+        <p><a href="https://www.ubuntu.com/support">进一步了解 Ubuntu Advantage&nbsp;›</a></p>
```
Comment (anthonydillon) [inline templates/server/index.html]: External link

File: templates/server/index.html [hunk 12]
```diff
@@ -13,543 +13,480 @@
...
-      <h3>裸机或云端都能实现出色速度与简洁性</h3>
-      <p>Ubuntu带来新的快速部署工具，可显著加快裸机安装服务器实例的速度。我们的服务编排工具 Juju，能让裸机或云端的大数据服务部署变得出人意料的简单。正因如此，10gen、Cloudera、Couchbase、DataStax、Hortonworks、LexisNexis和Map-R等厂商都选择与我们合作。</p>
-      <p><a href="/cloud">进一步了解Ubuntu Cloud&nbsp;›</a></p>
+  <div class="p-strip is-deep" style="padding-bottom: 2rem;">
+    <div class="row">
+      <div class="col-8">
+        <h2 id="built-for-big-data">为大数据和云而生</h2>
+        <h3>裸机或云端都能实现出色的速度和简洁性</h3>
+        <p>Ubuntu 带来新的快速部署工具，可显著加快在裸机上安装服务器实例的速度。我们的服务编排工具 Juju，能让裸机或云端的大数据服务部署出乎意料地简单。也正是这个原因，10gen、Cloudera、Couchbase、DataStax、Hortonworks、LexisNexis 和 Map-R 等厂商都选择与我们合作。</p>
+        <p><a href="https://www.ubuntu.com/cloud">进一步了解 Ubuntu Cloud&nbsp;›</a></p>
```
Comment (anthonydillon) [inline templates/server/index.html]: External

**Comment by anasereijo** [comment]
design suggestions:

- [ ] external links shouldn't be orange:
` p-link--external` should be `#007aa6` as per our vanilla pattern 

- [ ] 18.04 LTS 新增功能 and 18.10 新增功能 sections:
can we separate both releases? I suggest copy the style we have on [ubuntu.com](https://www.ubuntu.com/desktop/developers). See below:
screenshot 2019-02-13 at 10 26 11

- [ ]  支持您的各类硬件和软件 muted heading:
something went wrong (`p-muted-heading` and links are left aligned). Please change it to look like what we [currently have online](https://cn.ubuntu.com/server)?

_how it looks like now:_

screenshot 2019-02-13 at 10 34 38


_how it should look like [(online version)](https://cn.ubuntu.com/server)_

screenshot 2019-02-13 at 10 33 51

- [ ] 可在任何地方部署, 为大数据和云而生, 实现虚拟化和容器化的捷径 sections:
same changes as above

**Comment by zprood** [comment]
@pmahnke Thanks, I will take a look for all the links.

File: templates/cloud/index.html [hunk 13]
```diff
@@ -322,7 +322,7 @@ <h2>在公有云上使用Ubuntu</h2>
...
       <figure id="ubuntu1-pie" class="ubuntu1-pie"><svg width="308" height="308" viewBox="0 0 310 310"><g transform="translate(155,155)"><path d="M9.429780353434619e-15,-154A154,154 0 1,1 -90.5189288530409,124.58861713374188L-45.25946442652045,62.29430856687094A77,77 0 1,0 4.7148901767173095e-15,-77Z" class="ubuntu" style="stroke: rgb(174, 167, 159); stroke-width: 1; stroke-dasharray: 3, 3;"></path><path d="M-90.5189288530409,124.58861713374188A154,154 0 0,1 -2.8289341060303857e-14,-154L-1.4144670530151929e-14,-77A77,77 0 0,0 -45.25946442652045,62.29430856687094Z" class="other" style="stroke: rgb(174, 167, 159); stroke-width: 1; stroke-dasharray: 3, 3;"></path><text y="-5" class="ubuntu-label"><tspan x="85.32943725585938" dy="0">ubuntu</tspan><tspan x="88.422607421875" dy="20">(60%)</tspan></text><text y="-5" class="other-label"><tspan x="-133.09738159179688" dy="0">other</tspan><tspan x="-132.77914428710938" dy="20">(40%)</tspan></text></g></svg></figure>
-      <script src="https://assets.ubuntu.com/v1/3c99518b-d3-version3.5.6.min.js"></script>
+      <script src="{{ ASSET_SERVER_URL }}3c99518b-d3-version3.5.6.min.js"></script>
```
Comment (anthonydillon) [inline templates/cloud/index.html]: Does cloudinary do js files?

File: templates/server/index.html [hunk 14]
```diff
@@ -4,552 +4,483 @@
...
-  <h2 class="p-muted-heading u-align-text--center">我们的部分大数据合作伙伴</h2>
-  <ul id="cloud-partners" class="p-inline-images">
+        <h2 class="p-muted-heading u-align--center">我们的部分大数据合作伙伴</h2>
+        <ul id="cloud-partners" class="p-inline-images">
 
-      <li class="p-inline-images__item">
-        <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/aadbed62-DataStax.png" alt="DataStax">
-      </li>
+          <li class="p-inline-images__item">
+            <a href="https://www.mapr.com/"><img class="p-inline-images__logo" src="https://assets.ubuntu.com/sites/ubuntu/1149/u/img/logos/partners/mapr_logo.jpg" style="max-width: 144px;" width="144" alt="MapR" /></a>
```
Comment (anthonydillon) [inline templates/server/index.html]: This could also cloudinary

File: templates/server/index.html [hunk 15]
```diff
@@ -4,552 +4,483 @@
...
+          </li>
 
-      <li class="p-inline-images__item">
-        <img class="p-inline-images__logo" src="https://assets.ubuntu.com/sites/ubuntu/1121/u/img/logos/partners/Cloudera.png" alt="Cloudera">
-      </li>
+          <li class="p-inline-images__item">
+            <a href="https://www.cloudera.com/"><img class="p-inline-images__logo" src="https://assets.ubuntu.com/sites/ubuntu/1121/u/img/logos/partners/Cloudera.png" style="max-width: 144px;" width="144" alt="Cloudera" /></a>
```
Comment (anthonydillon) [inline templates/server/index.html]: And this one

File: templates/server/index.html [hunk 16]
```diff
@@ -4,552 +4,483 @@
...
-      </li>
+          <li class="p-inline-images__item">
+            <a href="http://www.datastax.com/"><img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}aadbed62-DataStax.png" style="max-width: 144px;" width="144" alt="DataStax" /></a>
+          </li>
 
-      <li class="p-inline-images__item">
-        <img class="p-inline-images__logo" src="https://assets.ubuntu.com/sites/ubuntu/1121/u/img/logos/partners/10genLogo_1.png" alt="MongoDB">
-      </li>
+          <li class="p-inline-images__item">
+            <a href="http://www.couchbase.com/"><img class="p-inline-images__logo" src="https://assets.ubuntu.com/sites/ubuntu/1121/u/img/logos/partners/couchbase_logo_final_SCALED___.png" style="max-width: 144px;" width="144" alt="Couchbase" /></a>
```
Comment (anthonydillon) [inline templates/server/index.html]: This one too

File: templates/cloud/index.html (hunk 13 above)
Comment (pmahnke) [reply templates/cloud/index.html]: no

**Comment by anasereijo** [comment]
awesome!

**Pull request merged**